pom.xml

```diff
@@ -92,6 +92,11 @@
             <artifactId>bitcoinj-core</artifactId>
             <version>0.14.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.7.5</version>
+        </dependency>
```

Comment:
I'm not sure this dependency should be introduced into **libdohj** -- See my comment in the "Conversation" for this pull-request about creating a separate NamecoinJ  library.